src/server.rs

```diff
@@ -76,6 +84,24 @@ async fn handle_measurement_submit(
       let body_str = from_utf8(&body)?.trim();
       let bincode_msg = base64_engine::STANDARD.decode(body_str)?;
       parse_message(&bincode_msg)?;
+
+      if let Some(min_revision) = state.min_revision_map.get(channel_name) {
+        let req_revision = request
+          .headers()
+          .get(HeaderName::from_static(REVISION_HEADER))
+          .map(|v| {
+            v.to_str()
+              .unwrap_or_default()
+              .parse::<usize>()
+              .unwrap_or_default()
```

Comment:
Just to confirm, this means requests without the revision header will be treated as if they sent `Brave-Constellation-Rev: 0` so we're not necessarily excluding all currently-deployed clients?

Reply:
correct